assets/dev/js/editor/elements/views/container.js

```diff
@@ -181,16 +181,35 @@ const ContainerView = BaseElementView.extend( {
 	},
 
 	/**
-	 * Add a `Save as Template` button to the context menu.
+	 * Insert a new container inside an existing container.
+	 *
+	 * @returns {void}
+	 */
+	addNewContainer() {
+		/* Check if the current container has a parent container */
+		const containerAncestry = this.getContainer().getParentAncestry(),
+			containerParentSelector = ( 'container' !== containerAncestry[ 1 ].type ) ? this.getContainer() : this.getContainer().parent;
```

Comment:
I would change the variable name, since what is returned is a Container instance, not a selector. I would change it to `targetContainer`, or `targetParentContainer`, or something of the sort.

Reply:
Also why do you check `containerAncestry[ 1 ]` and not `containerAncestry[ 0 ]`?

Reply:
Hi Udi, 

Thank you for your feedback about naming conventions. I have updated it.

Ancestry check: the current element is defined as the first parent inside the function. I assume the reasoning behind that is: in that way you can easily remember the original definition of the const/variable. The first parent is therefore always element [ 1 } and not [ 0 ].

See: https://github.com/elementor/elementor/blob/f2da4a0ac3eae4f2ae20a38065629ea44f662f94/assets/dev/js/editor/container/container.js#L286